app/views/orgs/_profile_form.html.erb

```diff
@@ -3,6 +3,9 @@
 <%
   shared_links_tooltip = _('Links will be displayed next to your organisation\'s logo')
   org_config_info_tooltip =  _('This information can only be changed by a system administrator. Contact the Help Desk if you have questions or to request changes.')
+  org_types_tooltip = _(
+    'This information can only be changed by a system administrator. Contact %{support_email} if you have questions or to request changes.' % { support_email: Rails.configuration.x.organisation.email}
+     )
```

Comment:
I think this should be fine. But we should just remember it if we have problems with translation